docs/usage/configuration.md

```diff
@@ -111,22 +111,30 @@ unused_ignore_directives = "warn"
 Each module listed under the `modules` key above can accept the following attributes:
 
 - `path` the Python import path to the module (e.g. `a.b` for `<root>/a/b.py`)
-!!! note
+
+    !!! note
         Glob patterns are allowed. The pattern `"libs.**"` would define the default configuration for any module under the `libs` namespace.<br/><br/>This can be overridden for specific modules by defining them later in the file using a concrete pattern like `"libs.module"`.
-!!! note
-        A module can also define `paths` as a shorthand for multiple module definitions. This allows specifying allowed dependencies and other attributes as a group.<br></br><br></br>Example: `paths = ["a.b", "a.c"]`
-    - `depends_on` a list of module paths which this module can import from
-!!! note
+
+    !!! note
+        A module can also define `paths` as a shorthand for multiple module definitions. This allows specifying allowed dependencies and other attributes as a group. Example: `paths = ["a.b", "a.c"]`
+
+- `depends_on` a list of module paths which this module can import from
+
+    !!! note
         Glob patterns are allowed. The pattern `"libs.**"` would allow dependencies on any module under the `libs` namespace.
-!!! note
+
+    !!! note
         Omitting the `depends_on` field means the module will be allowed to import from any other module. However, it will still be subject to those modules' [public interfaces](#interfaces).
-    - `cannot_depend_on` a list of module paths which this module cannot import from
-!!! note
+
+- `cannot_depend_on` a list of module paths which this module cannot import from
+
+    !!! note
         This takes precedence over `depends_on`. In other words, if `cannot_depend_on = ["module"]`, then `depends_on = ["module"]` will have no effect.
-    - `layer` (optional) the [**layer**](#layers) which holds this module
-    - `visibility` (optional) a list of other modules which can import from this module
-    - `utility` (default: `false`) marks this module as a **Utility**, meaning all other modules may import from it without declaring an explicit dependency
-    - `unchecked` (default: `false`) marks this module as [**unchecked**](unchecked-modules.md), meaning Tach will not check its imports
+
+- `layer` (optional) the [**layer**](#layers) which holds this module
+- `visibility` (optional) a list of other modules which can import from this module
+- `utility` (default: `false`) marks this module as a **Utility**, meaning all other modules may import from it without declaring an explicit dependency
+- `unchecked` (default: `false`) marks this module as [**unchecked**](unchecked-modules.md), meaning Tach will not check its imports
 
 !!! note
         Tach also supports [deprecating individual dependencies](deprecate.md).
```

Comment:
all the other notes are messed up too. i think they're indented too far which causes the contents to incorrectly appear as codeblocks

```suggestion
    Tach also supports [deprecating individual dependencies](deprecate.md).
```